crates/driver/src/domain/competition/solution/scoring.rs

```diff
@@ -93,16 +100,15 @@ impl Trade {
     /// fees have been applied.
     ///
     /// Denominated in SURPLUS token
-    fn surplus(&self) -> Option<eth::Asset> {
+    fn surplus_over_limit_price(&self, limit: Amounts) -> Option<eth::Asset> {
```

Comment:
One of the things we started doing with DDD is try to keep method names as concise as possible. So instead of calling something `surplus_over_limit_price`, assuming that it's not very ambiguous, would keep the name `surplus` and specify via function arguments and method comments what the method does.

I don't feel very strongly (historically I was in favour of writing a few more keystrokes to get more clarity), but I do think that the more recent code using short names and module paths to give additional context, looks very nice.

Reply:
nit: I would agree that just "surplus" is fine instead of "surplus_over_limit_price"